src/fragments/lib/ssr/js/getting-started.mdx

```diff
@@ -1,5 +1,11 @@
 To work well with server-rendered pages, Amplify JS requires slight modifications from how you would use it in a client-only environment.
 
+<Callout>
+
+The SSR implementation from Amplify will only work with server side functions that have a `req` object such as `getServerSideProps` using NextJS. Server side functions such as `getStaticPaths` and `getStaticProps` will not work.
```

Comment:
It's not that it won't work. You just won't be able to use client-side auth cookies on the server to perform authenticated requests as the user that's signed in on the client.

Reply:
And it's not an Amplify limitation per se. It's not possible to implement this without access to the `req` object.

Reply:
updated now! can you check the latest commit